mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala

```diff
@@ -607,6 +607,73 @@ class LogisticRegressionSuite
     assert(model2.coefficients ~= coefficientsR2 relTol 1E-2)
   }
 
+
+  test("an extra large example for review only") {
```

Comment:
This test is for review only, to show an example on a larger data set. Will remove if merged.